docs/electronics/skrpro.md

```diff
@@ -0,0 +1,117 @@
+
+![!Series](https://www.v1engineering.com/wp-content/uploads/2020/07/Series-skr-labels-scaled.jpg){: width="400"}
+
+### Dual End Stops (Exrtuders =0)
+
+If the firmware is set for extruders=0 then E0 becomes X1 and E1 becomes Y2. (or LR would be E0=Y2 E1=Z2)
+
+![!dual 0](https://www.v1engineering.com/wp-content/uploads/2020/07/dual2-scaled.jpg){: width="400"}
+
+### Dual End Stops (Extruders =1)
+
+For extruders=1 E0 is used and then E1 becomes X1 and E2 become Y2. (or LR would be E1=Y2 and E2=Z2)
+
+![!dual 1](https://www.v1engineering.com/wp-content/uploads/2020/07/Dual-Ex-1-scaled.jpg){: width="400"}
+
+
+### Reversing a stepper
+
+If your stepper is moving the wrong way you can power down and flip the plug over to change the direction of rotation. If you are using the series wiring harness flipping the board end changes both stepper, or changing one stepper in the chain reverse that one only.
+
+![!flip 1](https://www.v1engineering.com/wp-content/uploads/2020/07/flip1-scaled.jpg){: width="400"}
+![!flip 2](https://www.v1engineering.com/wp-content/uploads/2020/07/flip2-scaled.jpg){: width="400"}
+
+## Screen TFT35 V3 E3
+
+This screen was chosen because it had a knob. We can use it from both Touchscreen mode and Classic mode, simultaneously. **Switching between modes is as easy and pressing the knob in for 3 seconds.**
+
+![!screen](https://www.v1engineering.com/wp-content/uploads/2020/07/lcd-touch-Labels-scaled.jpg){: width="400"}
+
+You do not need to use both, pay attention to the direction the wires are facing when connecting them.
+
+## Touch plate
+
+All the V1 firmware is ready for a touch plate. Easy as plugging into the Z min port.
+
+![!Probe](https://www.v1engineering.com/wp-content/uploads/2020/07/Z-probe-scaled.jpg){: width="400"}
+
+
+## Dual End Stops, End Stops
+
+If you are running the dual endstop firmware you will need to have your endstops wired in Normally Closed (NC). You will plug into the Signal and Ground pins, Do not use the + pins.
+
+![!endstops](https://www.v1engineering.com/wp-content/uploads/2020/07/endstops-scaled.jpg){: width="400"}
+
+!!!Note
+    Do not use the + (positive) pins or you will ruin your SKR Pro board.
+
+
+## Firmware
+
+The SKR Pro and TFT screen both can use a bin file to re-flash the firmware. This is as simple as placing some files directly on the memory card and rebooting them.
+
+### SKR Bin
+
+Links needed.
+
+### TFT Files
+
+Links and screenshot needed.
+
+## Compile your own (source)
+
+If you want to make any changes you will need to compile form the source files using Platform.io. Here is a basic walk through, [Docs P.io](../learn/platformio.md).
+
+The V1 Engineering version of the SKR Pro firmware can be found on [github](https://github.com/Allted/Marlin), there are two versions, series and MPCNC dual endstop.
```

Comment:
Oops. I edited your main menu in CHOOSE_VERSION and it let me commit straight in, without a PR. Sorry.